src/metadata/mwdb_uploads.py

```diff
@@ -0,0 +1,64 @@
+import re
+import urllib.parse
+
+from metadata import Metadata
+from mwdblib import Malwarecage  # type: ignore
+from typing import List, Any
+
+
+class MalwarecageUploadsMetadata(Metadata):
+    """
```

Comment:
:+1: docstrings in a standard format are a great idea

Reply:
^^